src/annotator/plugin/test/document-test.js

```diff
@@ -64,11 +64,11 @@ describe('DocumentMeta', function () {
         <meta name="citation_title" content="Foo">
         <meta name="citation_pdf_url" content="foo.pdf">
         <meta name="dc.identifier" content="doi:10.1175/JCLI-D-11-00015.1">
-        <meta name="dc:identifier" content="foobar-abcxyz">
+        <meta name="dc:Identifier" content="foobar-abcxyz">
```

Comment:
Rather than modifying an existing test I think it would be better to add a separate test to cover this change more explicitly. As it is this particular test is now checking two things - handling of `:` vs `.` and lower-casing of the name.

Reply:
I agree. I added a separated test.